ods_ci/tests/Tests/400__ods_dashboard/430__data_science_pipelines/434__data-science-pipelines-ui.robot

```diff
@@ -61,6 +61,8 @@ Verify User Can Create, Run and Delete A DS Pipeline From DS Project Details Pag
     Verify Pipeline Run Deployment Is Successful    project_title=${PRJ_TITLE}
     ...    workflow_name=${workflow_name}
     Delete Pipeline Run    ${PIPELINE_TEST_RUN_BASENAME}    ${PIPELINE_TEST_NAME}
+    Delete Pipeline    ${PIPELINE_TEST_NAME}
```

Comment:
you need to test deletion of the pipeline server too

Reply:
Ok. It is out of the scope of this PR

Reply:
as you prefer Diego, but please plan to raise another one to complete the test case ODS-2226